### PR TITLE
BaSP-MR-04: update post-delete for the problem

### PR DESCRIPTION
**Pull request by GinoLucci1**
finish part post and delete, il wait for review and possible conflicts

### CODE REVIEW

**Review by Bigzufi — CHANGES_REQUESTED** [review]
You need to resolve the conflicts.

File: src/models/activity.js
Comment (Bigzufi) [inline src/models/activity.js:1]: The name of this file must be in PascalCase, so it has to be Activity.js

File: src/controllers/activity.js
Comment (Bigzufi) [inline src/controllers/activity.js:1]: As a suggestion, the response messages should follow the format given in the problem statement

File: src/models/activity.js
Comment (GinoLucci1) [reply src/models/activity.js:1]: It is written like this in local, the change from lowercase to uppercase should appear in git

**Review by Bigzufi — APPROVED** [review]
Looks good to me

**Pull request merged**